doc/source/conf.py

```diff
@@ -162,7 +162,7 @@
 # General information about the project.
 project = "pandas"
 # We have our custom "pandas_footer.html" template, using copyright for the current year
-copyright = f"{datetime.now().year}"
+copyright = f"{datetime.now().year} "
```

Comment:
Can we use the comma ending instead of the white space?

Reply:
You could but that comma would presumably appear in the rendered documentation - my reason for using a space here and removing it from the other file was to not change the rendered documentation

Reply:
I think changing the rendered docs is okay if it simplifies the changes

Reply:
That simplification is literally "don't remove one space", and given that the code is already written, changing it now is more work not less.  But this isn't worth arguing over, so whatever.